core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -200,8 +232,15 @@ protected Map<String, ByteBuffer> onLeaderElected(
       error = SchemaRegistryProtocol.Assignment.DUPLICATE_URLS;
     }
 
-    // All members currently receive the same assignment information since it is just the leader ID
     Map<String, ByteBuffer> groupAssignment = new HashMap<>();
+    if (existingLeaderKafkaId != null) {
```

Comment:
Can we also add && existingLeaderIdentity != null to ensure if this is set to null in the future, we don't hit any issue.

Reply:
if `existingLeaderKafkaId != null` then `existingLeaderIdentity != null` but I'll add it anyway.